src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -2327,6 +2331,229 @@ protected override void HandleSetBreakpointsRequestAsync(IRequestResponder<SetBr
             responder.SetResponse(response);
         }
 
+        protected override void HandleDataBreakpointInfoRequestAsync(IRequestResponder<DataBreakpointInfoArguments, DataBreakpointInfoResponse> responder)
+        {
+            if (responder.Arguments.Name == null)
+            {
+                responder.SetError(new ProtocolException("SetDataBreakpointRequest failed: Missing 'breakpoints'."));
+                return;
+            }
+
+            DataBreakpointInfoResponse response = new DataBreakpointInfoResponse();
```

Comment:
>DataBreakpointInfoResponse [](http://example.com/codeflow?start=12&length=26)

Wrap body in try/catch (all of these methods should, but I wouldn't advise fixing that in this PR)